index.html

```diff
@@ -195,6 +195,45 @@ <h3 class="aboutme-techs-title">Skills</h3>
          </ul>
       </li>
     </ul>  
+ </section>
+ <section class="contact-form">
+   <header class="contact-logo">
+    <img  src="./images/image_geometry_1.png" alt="logo image">
+   </header>
+   <div class="contact-card">
+    <h2 class="contact-title">Get Started</h2>
+    <p class="contact-para">I'm always interested in hearing about new projects, 
+      so if you'd like to chat please get in touch.</p>
+      <form action="https://formspree.io/f/xqkovbew"
+      method="POST" class="contact">
+
+        <label for="name">Your name</label>
+        <input type="text" id="fname" maxlength="30" required>
+        <label for="email">Your email</label>
+        <input type="text" id="mail" required>
```

Comment:
Hi, thank you for your submission. I have reviewed your code and found a minor issue. Here is a suggestion:

- Please use the input type of email instead of text for the email field. This will help to validate the user input and ensure that it is a valid email address. It will also provide a better user experience by showing the appropriate keyboard and autofill options on different devices. You can learn more about the email input type here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email

I have provided a corrected version of your code below. Please review it and make the necessary change. If you have any questions or doubts, please let me know.

`<input type="email" id="mail" required>`